plugins/woocommerce/includes/emails/class-wc-email.php

```diff
@@ -559,18 +563,20 @@ public function style_inline( $content ) {
 			wc_get_template( 'emails/email-styles.php' );
 			$css = apply_filters( 'woocommerce_email_styles', ob_get_clean(), $this );
 
-			$emogrifier_class = 'Pelago\\Emogrifier';
+			$css_inliner_class = CssInliner::class;
 
-			if ( $this->supports_emogrifier() && class_exists( $emogrifier_class ) ) {
+			if ( $this->supports_emogrifier() && class_exists( $css_inliner_class ) ) {
 				try {
-					$emogrifier = new $emogrifier_class( $content, $css );
+					$css_inliner = CssInliner::fromHtml( $content )->inlineCss( $css );
+
+					do_action( 'woocommerce_emogrifier', $css_inliner, $this );
 
-					do_action( 'woocommerce_emogrifier', $emogrifier, $this );
+					$domDocument = $css_inliner->getDomDocument();
```

Comment:
```suggestion
					$dom_document = $css_inliner->getDomDocument();
```

Reply:
The consequences of reckless copypaste. Fixed!